### ISSUE TITLE
Lidarr

### ISSUE THREAD

**Issue by AnythingOldSkewl** [coder8338/appbox_installer#38]
This is more of a question than an issue I guess. The problem is, Lidarr says I'm using the latest version (0.8.1.2135/Apr 18 2021). Yet, I get a message in the "Health status" section, to please upgrade to the .NET Core version of Lidarr. For now, Lidarr appears to be working fine the way it is. However, I also don't know how urgent or necessary this upgrade is.

**Comment by rollecode** [comment]
Just noticed this with Lidarr:

```
$ sudo tail -f /var/log/appbox/lidarr/current

2021-09-24 14:16:00.052938212  rm: cannot remove '/home/appbox/.config/Lidarr/lidarr.pid': No such file or directory
2021-09-24 14:16:00.100757251  
2021-09-24 14:16:00.100763168  Unhandled Exception:
2021-09-24 14:16:00.100764917  System.TypeLoadException: Could not resolve type with token 01000011 from typeref (expected class 'System.Net.Sockets.SocketException' in assembly 'System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=xxxxxxxxxxxxxxxx')
2021-09-24 14:16:00.100766667  [ERROR] FATAL UNHANDLED EXCEPTION: System.TypeLoadException: Could not resolve type with token 01000011 from typeref (expected class 'System.Net.Sockets.SocketException' in assembly 'System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=xxxxxxxxxxxxxxxx')
```

Related to version mismatch AND .NET Core version upgrade I did manually.

Fixed this in this PR #50.

**Comment by rollecode** [comment]
It seems it doesn't run correctly even with the switch to new version...

Running in screen directly with command `./Lidarr` works but not when in `/run/s6/services/lidarr/run`...

Any ideas?

**Comment by coder8338** [comment]
Closed by #50
